TMSpart2/AdditionalExercisesLesson16.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+
+namespace TMSpart2
+{
+    public static class AdditionalExercisesLesson16
+    {
+        public static int CountWords(string sentence)
+        {
+            try
+            {
+                var counter = sentence.Split(' ').Select(a => a.Trim()).ToArray().Where(x => x != "").Count();
```

Comment:
you may use Split with StringSplitOptions.RemoveEmptyEntries options without using Trim. Also using ToArray() is odd in this situation.
 example: var counter = sentence
 .Split(' ',StringSplitOptions.RemoveEmptyEntries)
 .Select(a => a)
 .Where(x => x != "")
 .Count();

Reply:
Couldn't use StringSplitOptions.RemoveEmptyEntries parameter - SV underlines it. But it works without it just
var counter = sentence.Split(' ').Where(x=>x !="").Count(); - doesn't count empty entries and looks like it's enough.
Thank you